test/test_audio.py

```diff
@@ -0,0 +1,155 @@
+import os
```

Comment:
Nice TDD work in this file! I like it.

Reply:
Thanks! Yes, it was nice to have a bite-sized chunk to work on properly